src/components/sidenav/sidenav.js

```diff
@@ -229,6 +230,7 @@ function SidenavDirective($mdMedia, $mdUtil, $mdConstant, $mdTheming, $animate,
     var lastParentOverFlow;
     var triggeringElement = null;
     var promise = $q.when(true);
+    var isBackdropDisabled = !!attr.mdDisableBackdrop;
```

Comment:
@ThomasBurleson In general validations for attributes are not good enough (for example here only adding `md-disable-backdrop` won't work. I'd really like to update a few attributes to validate `false` and `true` and also an empty attribute. This will be easy if we have our util method, which is created in #6803